app/models/application_record.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def aplication
```

Comment:
__[Misspell]__

"aplication" is a misspelling of "application"


<!-- Comment by [Sider](https://sider.review) -->